src/compiler/tsbuild.ts

```diff
@@ -1788,31 +1788,25 @@ namespace ts {
 
         let reportQueue = true;
         let successfulProjects = 0;
-        let errorProjects = 0;
         while (true) {
             const invalidatedProject = getNextInvalidatedProject(state, buildOrder, reportQueue);
             if (!invalidatedProject) break;
             reportQueue = false;
             invalidatedProject.done(cancellationToken);
-            if (state.diagnostics.has(invalidatedProject.projectPath)) {
-                errorProjects++;
-            }
-            else {
-                successfulProjects++;
-            }
+            if (!state.diagnostics.has(invalidatedProject.projectPath)) successfulProjects++;
         }
 
         disableCache(state);
         reportErrorSummary(state, buildOrder);
         startWatching(state, buildOrder);
 
         return isCircularBuildOrder(buildOrder) ?
-            ExitStatus.ProjectReferenceCycle_OutputsSkupped :
-            errorProjects ?
-                successfulProjects ?
-                    ExitStatus.DiagnosticsPresent_OutputsGenerated :
-                    ExitStatus.DiagnosticsPresent_OutputsSkipped :
-                ExitStatus.Success;
+                 ExitStatus.ProjectReferenceCycle_OutputsSkipped

```

Comment:
Can you please use same pattern that either puts `?` and `:` at the end of the line or beginning of the line.

Reply:
Done.